### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by targi**
I needed to use Django-Feeback on a site translated into Russian, so I added this translation. Everything displayed to users is translated. Column titles displayed in admin like URL are left as is. Both `.mo` and `.po` files are included.

### CODE REVIEW

File: feedback/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (jaredly) [inline feedback/locale/ru/LC_MESSAGES/django.po]: maybe update this comment to say "Russian translation" or something?

**Comment by targi** [comment]
Thanks for the feedback. I have filled in the header according to the recommendations I found [here](http://localization-guide.readthedocs.org/en/latest/guide/project/howto.html). While I was at it, I also copied-and-pasted strings that didn't require translation.

If there's anything else that needs to be fixed, please point it out.

File: feedback/locale/ru/LC_MESSAGES/django.po (hunk 1 above)
Comment (targi) [inline feedback/locale/ru/LC_MESSAGES/django.po]: Fixed. See my comment on the pull request.

**Comment by jaredly** [comment]
awesome, thanks!

**Pull request merged**